### PR TITLE
hide accel related items when accel disabled

### LINKED ISSUE #2597: Hide accelerometer dependent items when accel is disabled
Following items probably shall be hided when accelerometer is disabled:

- FPV camera angle on configuration tab
- Acro trainer angle limit on PIDs tab
- FPV angle mix and Launch control on Modes tab
- Angle related elements on OSD tab as an option

anything else?

### PR DESCRIPTION
**Pull request by mituritsyn**
partially resolves #2597

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by mituritsyn** [comment]
> This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs within a week.

пыщ-пыщ

**Comment by haslinghuis** [comment]
@mituritsyn please also solve the other cases you mentioned.

**Review by blckmn — APPROVED** [review]

**Pull request merged**
